src/meshapi/migrations/0031_create_meshdb_ro.py

```diff
@@ -0,0 +1,87 @@
+# Generated by Django 4.2.15 on 2024-09-18 01:53
```

Comment:
To be clear: I do support doing this sort of thing with a migration, but I'll note that in #506 you had me take my custom SQL migration and move it into the main application code so that we don't have to carefully extract it when we nuke and re-generate the migrations prior to launch.

I think this is not that bad to extract, but we will have to do it

Reply:
Can you jog my memory about waht specific migration this was? Was it this? 

https://github.com/nycmeshnet/meshdb/pull/506/files/89603f171137062468c8e06d89523d754f1d2c0d#diff-543b583d981a7222ba8ac3f5136152fe92754cb704a303e21c55fc59d742cd69

Reply:
This the migration in question:
https://github.com/nycmeshnet/meshdb/pull/506/files#diff-64acc684bdd1b6f78227e5b50f104b3a7fb2bc79aec6ea69eff92678583512acR11

and this is the line that gets us out of needing it to be present:
https://github.com/nycmeshnet/meshdb/pull/506/commits/14d558b623d718d9b944b9f92c0062f8e4e5f594#diff-297eadf0b00ee217b77ccf35aff6ebc6a5b398624387be526b67d9bf9fbb054eR21

Reply:
This the migration in question:
https://github.com/nycmeshnet/meshdb/pull/506/files#diff-64acc684bdd1b6f78227e5b50f104b3a7fb2bc79aec6ea69eff92678583512acR11

and this is the line that gets us out of needing it to be present:
https://github.com/nycmeshnet/meshdb/pull/506/commits/14d558b623d718d9b944b9f92c0062f8e4e5f594#diff-297eadf0b00ee217b77ccf35aff6ebc6a5b398624387be526b67d9bf9fbb054eR21

Reply:
Yeah I think a `manage.py` command, is really our only other option, and to be clear, I think this is the right place to do this. It's just evidence that we shouldn't generally plan to mess with migrations in the future